src/test/java/com/gooddata/report/ReportServiceIT.java

```diff
@@ -1,97 +0,0 @@
-/*
- * Copyright (C) 2004-2017, GoodData(R) Corporation. All rights reserved.
- * This source code is licensed under the BSD-style license found in the
- * LICENSE.txt file in the root directory of this source tree.
- */
-package com.gooddata.report;
```

Comment:
Do we really want to remove all the tests? The service is still there, deprecated or not...

Reply:
added simple mocked tests for ReportService